utils/state/node.go

```diff
@@ -355,6 +358,7 @@ func addNodeDetailsCalls(contracts *NetworkContracts, mc *multicall.MultiCaller,
 
 	// Saturn
 	if contracts.isSaturnDeployed() {
+		mc.AddCall(contracts.RocketNodeStaking, &details.EthBorrowed, "getNodeETHBorrowed", address)
```

Comment:
Don't we need getNodeEthBonded here as well?

Reply:
Added, along with other new calls from RocketNodeStaking that might be needed